data/human/human missions.txt

```diff
@@ -6645,21 +6617,15 @@ mission "Southern Fiance 1"
 	on complete
 		payment
 		payment 10000
-		log "Minor People" "Tom and Carl" `Tom Ardwright was a marketer for the Syndicate, but moved south, with your help, to live with his government minister fiance Carl Chamberlain after war broke out.`
+		log "Minor People" "Tom and Carl" `Tom Ardwright was a marketer for the Syndicate, but moved south to live with his fiance Carl Chamberlain after war broke out.`
 		event "tom and carl's wedding" 60
-		event "carl elected governor" 150
-		event "tom and carl's child" 1500
 		conversation
-			`During the trip to <destination>, you had several conversations with Tom, on a wide range of topics. The most common topic was his fiance, named Carl Chamberlain, sharing dozens of pictures of them together. He also presented some of his own portfolio, as a marketer and designer for a division of Syndicated Systems - at least until he quit his job for this. "I'm not worried, though," he remarks. "Money's fun, of course, but it's nothing compared to what we can do down there on Cornucopia."`
-			`	The trip south seems to fly by, and soon you are standing at the foot of the ramp shaking Tom's hand and bidding him farewell as Carl runs across the tarmac toward you. You step back as they practically leap into each other's arms, reuniting in spite of these tumultuous times.`
-			`	When you're back on board, you see an incoming message that informs you <payment> have been transferred to your account, along with a heartfelt thank-you note from both men.`
+			`During the trip to <destination>, you had several conversations with Tom on a wide range of topics, including his fiance, Carl, as well as his job as a marketer and designer for a division of Syndicated Systems - at least until he quit. "It'll be a shame to see my work for the Syndicate be replaced, but I don't regret leaving. Carl's got big plans in store for <planet>, now that we aren't stuck with the Republic's shoestring budget, and I wouldn't miss them for the galaxy."`
+			label landing
+			`	The trip south seems to fly by, and soon you are standing at the foot of the ramp accepting Tom's fare of <payment> and bidding him farewell as Carl runs across the tarmac toward you. You step back as they practically leap into each other's arms, reuniting in spite of these tumultuous times.`
```

Comment:
What's with the label here? This conversation has two lines of text. A label between them seems very strange.

The second lines seems kind of strange, as well.
"The trip south seems to fly by," sounds like it's beginning the discussion of the trip, but we've actually already had a line discussing the trip.

Reply:
```suggestion
			`During the trip to <destination>, you had several conversations with Tom on a wide range of topics, including his fiance, Carl, as well as his job as a marketer and designer for a division of Syndicated Systems - at least until he quit. "It'll be a shame to see my work for the Syndicate be replaced, but I don't regret leaving. Carl's got big plans in store for <planet>, now that we aren't stuck with the Republic's shoestring budget, and I wouldn't miss them for the galaxy."`
			`	You stand at the foot of the ramp accepting Tom's fare of <payment> and bidding him farewell as Carl runs across the tarmac toward you. You step back as they practically leap into each other's arms, reuniting in spite of these tumultuous times.`
```